### PR TITLE
chore: deprecate PPC64/POWER builds

### PR DESCRIPTION
**Pull request by ShreyaLnuHpe**
## Ticket

[[RM-294](https://hpe-aiatscale.atlassian.net/browse/RM-294)]

## Description

Deprecate PPC64/POWER builds. Note: This is not just AgentRM specific, but will be deprecated for all RMs 

Potential Impact: Built for a single OSS [~3 years ago](https://determined-community.slack.com/archives/CV37XVAGL/p1617206888020800), haven’t heard of customers using it since. Users can’t use Determined on PPC64 architecture anymore 

Packages with our code do not support power pc. Ok to remove in Q3 / 0.33.1.

Current AgentRM deprecation document linked [here](https://hpe.sharepoint.com/:w:/r/teams/detai/_layouts/15/Doc.aspx?sourcedoc=%7BD5D9B85A-7371-42C5-AAF6-4DAD09D2EF98%7D&file=Pruning%20Under-utilized%20AgentRM%20Features%20(Needs%20Feedback%20for%20Commitment%20of%20Work).docx&action=default&mobileredirect=true).
No active customers use PPC64 architecture anymore. Hence it is safe to deprecate. [Here](https://hpe-aiatscale.slack.com/archives/C02PV33GSN5/p1710880143328809?thread_ts=1710863001.723729&cid=C02PV33GSN5) are more information about it.

## Test Plan

1. Tried to build an image and upload to circle CI. Images didn't have PPC64 OS arch 

## Checklist

- [X] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [X] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[DET-294]: https://hpe-aiatscale.atlassian.net/browse/DET-294?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[RM-294]: https://hpe-aiatscale.atlassian.net/browse/RM-294?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/release-notes/deprecate-ppc64 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+:orphan:
```
Comment (NicholasBlaskey) [inline docs/release-notes/deprecate-ppc64:1]: I think this needs to be a ```.rst``` file

**Review by NicholasBlaskey — APPROVED** [review]
looks great

LGTM

File: docs/release-notes/deprecate-ppc64.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Deprecations**
+
+-  Machine Architectures: Due to underutilized AgentRM features, support for PPC64/POWER builds is
```
Comment (kkunapuli) [inline docs/release-notes/deprecate-ppc64.rst:5]: I wouldn't have included "Due to underutilized AgentRM features".

Comment (ShreyaLnuHpe) [reply docs/release-notes/deprecate-ppc64.rst:5]: I have corrected the release note as per the discussion in this [PR](https://github.com/determined-ai/determined/pull/9470)

**Pull request merged**